includes/wc-product-functions.php

```diff
@@ -1310,3 +1266,27 @@ function wc_deferred_product_sync( $product_id ) {
 
 	$wc_deferred_product_sync[] = $product_id;
 }
+
+/**
+ * Add sorting index data for products.
+ */
+function wc_update_product_lookup_tables() {
+	global $wpdb;
+
+	$result = $wpdb->query(
+		"
+		INSERT IGNORE INTO {$wpdb->wc_product_sorting} (`product_id`, `price`, `min_price`, `max_price`, `average_rating`, `total_sales`)
+		SELECT posts.ID, MIN(meta1.meta_value), MIN(meta1.meta_value), MAX(meta1.meta_value), meta2.meta_value, meta3.meta_value
```

Comment:
`meta2.meta_value, meta3.meta_value` need to be added to the group by clause.

Reply:
Not sure I follow? Why - it's grouped by ID

Reply:
When I run the `INSERT` I get the following error

`ERROR 1055 (42000): Expression #4 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'meta2.meta_value' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by`